astrobee/launch/robot/MLP.launch

```diff
@@ -38,7 +38,7 @@
     <node pkg="nodelet" type="nodelet" args="manager"
           name="mlp_depth_cam" output="$(arg output)"/>
     <node pkg="nodelet" type="nodelet" args="manager"
-          name="mlp_mapper" output="$(arg output)" launch-prefix="nice -n 10"/>
+          name="mlp_mapper" output="$(arg output)" launch-prefix="nice -n 5"/>
```

Comment:
lowered this value due to the reports that the previous one would result in heartbeat and crashing problems